Programs/_bootstrap_python.c

```diff
@@ -64,6 +64,15 @@ main(int argc, char **argv)
     PyStatus status;
 
     PyConfig config;
+    PyPreConfig preconfig;
+    PyPreConfig_InitIsolatedConfig(&preconfig);
+    // Force utf8 encoding
+    preconfig.utf8_mode = 1;
+    status = Py_PreInitialize(&preconfig);
+    if (PyStatus_Exception(status)) {
+        Py_ExitStatusException(status);
+    }
+
```

Comment:
I recommend moving `config` down so it's clear the preconfig stuff isn't related:

```suggestion
    PyPreConfig preconfig;
    PyPreConfig_InitIsolatedConfig(&preconfig);
    // Force utf8 encoding
    preconfig.utf8_mode = 1;
    status = Py_PreInitialize(&preconfig);
    if (PyStatus_Exception(status)) {
        Py_ExitStatusException(status);
    }

    PyConfig config;
```